src/jvmMain/kotlin/me/devnatan/yoki/resource/container/ContainerResource.jvm.kt

```diff
@@ -1,21 +1,11 @@
 package me.devnatan.yoki.resource.container
 
-import io.ktor.client.HttpClient
-import io.ktor.client.call.body
-import io.ktor.client.request.accept
-import io.ktor.client.request.delete
-import io.ktor.client.request.get
-import io.ktor.client.request.head
-import io.ktor.client.request.parameter
-import io.ktor.client.request.post
-import io.ktor.client.request.preparePost
-import io.ktor.client.request.put
-import io.ktor.client.request.setBody
-import io.ktor.http.ContentType
-import io.ktor.http.HttpStatusCode
-import io.ktor.utils.io.ByteReadChannel
-import io.ktor.utils.io.jvm.javaio.toByteReadChannel
-import io.ktor.utils.io.readUTF8Line
+import io.ktor.client.*
+import io.ktor.client.call.*
+import io.ktor.client.request.*
+import io.ktor.http.*
+import io.ktor.utils.io.*
+import io.ktor.utils.io.jvm.javaio.*
```

Comment:
I believe that wildcards should not pass lint 🤔 Perhaps the configuration is wrong :/ if possible please disable wildcard imports in your IDE